Week3/homework/step2-6.js

```diff
@@ -4,13 +4,21 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function printArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr[i].length; j++) {
+      console.log(arr[i][j]);
+    }
+  }
 }
 
 function printArray3d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  let result = '';
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr[i].length; j++) {
+      result += arr[i][j].join('');
+    }
+  }
+  return console.log(result);
```

Comment:
There is not much point in return the result of a `console.log` statement: console.log returns `undefined`, i.e. it doesn't return anything. If you leave the `return` keyword everything still works.

Note that the version for the 2d array, with a simple console.log inside the inner loop, is sufficient to pass the test. When given a choice between a simpler solution and a more complex one that both achieve the same result, the simpler solution should take precedence.